CITATION.cff

```diff
@@ -0,0 +1,19 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Stephenson"
+  given-names: "Mark"
+  # orcid: "https://orcid.org/0000-0000-0000-0000"
```

Comment:
Should we add the ORCIDs?

Reply:
Sure, I should probably get around to making one. Feel free to send me yours